README.md

```diff
@@ -0,0 +1,85 @@
+# MITOC Website
+
+This repository contains the source code of https://mitoc.mit.edu. 
+
+All members of the MITOC community are welcome to suggest edits by opening pull requests!
+
+If you'd like to change anything on the website, you can:
+- Read the rest of this short document to figure out how things work
+- Suggest a change on a file by clicking on the "Edit this file" button (see screenshot below)
```

Comment:
I'd add a sentence here about the process (in re PRs). Maybe something like: `Suggest a change on a file by clicking on the "Edit this file" button (see screenshot below). This will open a Pull Request, which the webmasters will review. When the CR gets merged to master it will trigger an automatic deployment with CircleCI`.